internal/command/project_application_oidc_test.go

```diff
@@ -1209,6 +1218,7 @@ func TestCommandSide_ChangeOIDCApplicationSecret(t *testing.T) {
 								"",
 								domain.LoginVersionUnspecified,
 								"",
+								[]string{},
```

Comment:
Please add test to set the list of allowed scope prefixes and also to set the list to empty again.